docs/data/base/components/table-pagination/TableUnstyled.tsx

```diff
@@ -104,24 +104,33 @@ const rows = [
   createData('Oreo', 437, 18.0),
 ].sort((a, b) => (a.calories < b.calories ? -1 : 1));
 
-const Root = styled('div')`
+const grey = {
+  200: '#d0d7de',
+  800: '#32383f',
+  900: '#24292f',
+};
+
+const Root = styled('div')(
+  ({ theme }) => `
   table {
-    font-family: arial, sans-serif;
+    font-family: IBM Plex Sans, sans-serif;
```

Comment:
The font of the `select` doesn't match the rest of the component (it's not inherited).

Reply:
That's there to make the whole table use IBM Plex Sans (the select is native, right? So that wouldn't indeed pick it up).